linux_docker_resources/entry_point.sh

```
@@ -12,41 +12,49 @@ export ORIG_GID=$(echo $ORIGPASSWD | cut -f4 -d:)
+  echo "Initializing Git-LFS..."
+  sudo -H -u rosbuild -- git lfs install
+  echo "done."
+else
+  echo "Not installing git-lfs because we're on a non-x86_64 machine."
 fi
-echo "done."
 
 case "${CI_ARGS}" in
   *--connext*)
-    echo "Installing Connext..."
-    case "${CI_ARGS}" in
-      *--osrf-connext-debs*)
-        echo "Installing OSRF-built Connext debs..."
-        dpkg -i /tmp/librticonnextdds52_5.2.3-1_amd64.deb
-        dpkg -i /tmp/librticonnextdds52-dev_5.2.3-1_amd64.deb
-        dpkg -i /tmp/rticonnextdds-tools_5.2.3-1_amd64.deb
-        ;;
-      *)
-        echo "Installing Connext binaries off RTI website..."
-        python3 -u /tmp/rti_web_binaries_install_script.py /tmp/rti-installer.run /home/rosbuild
-        if [ $? -ne 0 ]
-        then
-          echo "Connext not installed correctly." >&2
-          exit 1
-        fi
-        mv /tmp/rti_license.dat /home/rosbuild/rti_license.dat
-        export RTI_LICENSE_FILE=/home/rosbuild/rti_license.dat
-        ;;
-    esac
-    echo "done."
+    if [ $ARCH = "x86_64" ]; then
+      echo "Installing Connext..."
+      case "${CI_ARGS}" in
+        *--osrf-connext-debs*)
+          echo "Installing OSRF-built Connext debs..."
+          dpkg -i /tmp/librticonnextdds52_5.2.3-1_amd64.deb
+          dpkg -i /tmp/librticonnextdds52-dev_5.2.3-1_amd64.deb
+          dpkg -i /tmp/rticonnextdds-tools_5.2.3-1_amd64.deb
+          ;;
+        *)
+          echo "Installing Connext binaries off RTI website..."
+          python3 -u /tmp/rti_web_binaries_install_script.py /tmp/rti-installer.run /home/rosbuild
+          if [ $? -ne 0 ]
+          then
+            echo "Connext not installed correctly." >&2
+            exit 1
+          fi
+          mv /tmp/rti_license.dat /home/rosbuild/rti_license.dat
+          export RTI_LICENSE_FILE=/home/rosbuild/rti_license.dat
+          ;;
+      esac
+      echo "done."
+    else
+      echo "Not installing Connext because we're on a non-x86_64 machine."
```

4047ddfbd89653366cb79ec7756557034903fb6e reverts my change to hide the failure to install Connext, replacing it with a more informative error message.

f91bbce19ab26d2cbfff4fd7f41b47831dbcb73c attempts to disable Connext specifically for the aarch64 job when started by the launcher. It's not clear to me how the `Parameterized Trigger Plugin` will merge the current build's params with one that's explicitly set; I have jobs in flight to figure that out.